src/main/scala/viper/gobra/frontend/info/implementation/typing/ExprTyping.scala

```diff
@@ -54,6 +61,7 @@ trait ExprTyping extends BaseTyping { this: TypeInfoImpl =>
         // imported members, we simply assume that they are wellformed (and were checked in the other package's context)
         case Some(_: ap.Constant) => noMessages
         case Some(_: ap.Function) => noMessages
+        case Some(_: ap.Closure) => violation(s"the name of a function literal should not be accessible from a different package")
```

Comment:
What does it mean exactly? Does it mean that you cannot assign a function to a variable if that function is declared in another package? If so, why do we have that restriction?

Reply:
No, we cannot access a function literal from another package (simply because they are declared within functions, so they are never visible from another package). This is not an additional restriction, just a sanity check - and to stop the compiler from warning about missing cases.

Functions from a different package can be assigned to variables, and they would match case (_: ap.Function)